setup.py

```diff
@@ -7,7 +7,7 @@
   name = 'archrepo2',
   version = archrepo2.__version__,
   packages = find_packages(),
-  install_requires = ['tornado>2.4.1', 'pyinotify', 'setuptools'],
+  install_requires = ['tornado>2.4.1', 'pyinotify', 'pyalpm', 'setuptools'],
```

Comment:
Is setuptools still required?

Reply:
After read the document of setuptools [[1]](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#install-requires)[[2]](https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/#install-requires-vs-requirements-files) and the [sampleproject](https://github.com/pypa/sampleproject/blob/db5806e0a3204034c51b1c00dde7d5eb3fa2532e/setup.py#L130C49-L130C49).

I think the ``setuptools`` on ``install_requires`` can be removed